packages/fxa-react/.storybook/webpack.config.js

```diff
@@ -1,26 +1 @@
-// https://storybook.js.org/docs/configurations/typescript-config/#setting-up-typescript-to-work-with-storybook-1
-module.exports = ({ config, mode }) => {
-  config.module.rules.push({
-    test: /\.(ts|tsx)$/,
-    loader: require.resolve('babel-loader'),
-    options: {
-      presets: [['react-app', { flow: false, typescript: true }]],
-    },
-  });
-  config.resolve.extensions.push('.ts', '.tsx');
-
-  config.module.rules.push({
-    test: /\.s[ac]ss$/i,
-    use:[
-     // Creates `style` nodes from JS strings
-    'style-loader',
-    // Translates CSS into CommonJS
-    'css-loader',
-    // Compiles Sass to CSS
-    'sass-loader'
-    ],
-  });
-  config.resolve.extensions.push('.scss', '.css');
-
-  return config;
-};
+module.exports = require("../../fxa-react/storybooks").customizeWebpackConfig;
```

Comment:
Looks like this is referencing from within fxa-react already, so could you just get away with `../storybooks`?